src/Microsoft.Management.Configuration.Processor/Public/ConfigurationSetProcessorFactory.cs

```diff
@@ -55,7 +55,9 @@ public IConfigurationSetProcessor CreateSetProcessor(ConfigurationSet set)
                 this.OnDiagnostics(DiagnosticLevel.Verbose, $"Creating set processor for `{set.Name}`...");
 
                 var envFactory = new ProcessorEnvironmentFactory(this.type);
-                var processorEnvironment = envFactory.CreateEnvironment(this);
+                var processorEnvironment = envFactory.CreateEnvironment(
+                    this,
+                    this.properties != null ? this.properties.Policy : ConfigurationProcessorPolicy.RemoteSigned);

```

Comment:
```suggestion
                    this.properties?.Policy ?? ConfigurationProcessorPolicy.RemoteSigned);
```
I'm pretty sure this is the fancy operator version.